### PR TITLE
14 homepage header

### LINKED ISSUE #14: Homepage Header
## Expected Behaviour

Cceate a Component for the Homepage Header.

## Checklist

_Detail the steps below to help us reproduce the problem you encountered._

- [x] Follow the component design in [figma](https://www.figma.com/design/TOCJzJFMXE7PL1dQyGWqXZ/midterm?node-id=193-51&t=3bsn1O4cipxo3ue9-1)
- [x] The component accepts a prop for the name
- [x] The component accepts a prop for an avatar image

### PR DESCRIPTION
**Pull request by multimediaschneider**
## Checklist

Confirm you have completed the following actions prior to submitting this PR.

- [x] There is an existing issue report for this PR.
- [x] I have created a feature branch.
- [x] My changes have been committed.
- x ] I have pushed my changes to the branch.

## Title

created the homepage header.

## Issue Resolution

_Tell us which issue this PR fixes._

Closes #14

## Proposed Changes

_List your proposed changes below._

-
-
-

## New or Changed Features

_Does this PR provide new or changed features or enhancements? If so, what is included in this PR?_

-
-
-

### CODE REVIEW

File: src/components/HomepageHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  userName,
+  userImage,
+}) => {
+  return (
+    <div className="flex w-screen h-10 bg-dark pl-5 pr-5 pt-8 text-white justify-between items-center">
+      <div className="flex-col">
+        <div className="pb-3 text-white-dimmed text-xs">
+          Welcome {userName} <span className="text-white">👋</span>
+        </div>
+        <div className="text-sm">Let's relax and watch a movie!</div>
```
Comment (dnmct) [inline src/components/HomepageHeader.tsx]: text should be bold

File: src/components/HomepageHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+export const HomepageHeader: React.FC<HomepageHeaderProps> = ({
+  userName,
+  userImage,
+}) => {
+  return (
+    <div className="flex w-screen h-10 bg-dark pl-5 pr-5 pt-8 text-white justify-between items-center">
+      <div className="flex-col">
+        <div className="pb-3 text-white-dimmed text-xs">
+          Welcome {userName} <span className="text-white">👋</span>
```
Comment (dnmct) [inline src/components/HomepageHeader.tsx]: text should be bold

File: src/components/HomepageHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  userName: string;
+  userImage: string;
+}
+
+export const HomepageHeader: React.FC<HomepageHeaderProps> = ({
+  userName,
+  userImage,
+}) => {
+  return (
+    <div className="flex w-screen h-10 bg-dark pl-5 pr-5 pt-8 text-white justify-between items-center">
```
Comment (dnmct) [inline src/components/HomepageHeader.tsx]: - doesn't need a height, the height is determined by the content
- The component has no padding

File: src/components/HomepageHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      <div className="flex-col">
+        <div className="pb-3 text-white-dimmed text-xs">
```
Comment (dnmct) [inline src/components/HomepageHeader.tsx]: to create space between element either use margin-bottom or gap on the flebox

File: src/components/HomepageHeader.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  userImage: string;
+}
+
+export const HomepageHeader: React.FC<HomepageHeaderProps> = ({
+  userName,
+  userImage,
+}) => {
+  return (
+    <div className="flex w-screen h-10 bg-dark pl-5 pr-5 pt-8 text-white justify-between items-center">
+      <div className="flex-col">
```
Comment (dnmct) [inline src/components/HomepageHeader.tsx]: this need flex -> display: flex;
flex-col is only setting the flex-direction

**Review by dnmct — CHANGES_REQUESTED** [review]

File: src/main.tsx
Comment (louziferr) [inline src/main.tsx:1]: Please remove unnecessary content from main.tsx

File: src/components/HomepageHeader.tsx
Comment (louziferr) [inline src/components/HomepageHeader.tsx:1]: Please rename to homepage-header.tsx

**Review by louziferr — CHANGES_REQUESTED** [review]
Changes need to be done before merging.

File: src/components/homepage-header.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+interface HomepageHeaderProps {
+  userName: string;
+  userImage: string;
+}
+
+export const HomepageHeader: React.FC<HomepageHeaderProps> = ({
```
Comment (louziferr) [inline src/components/homepage-header.tsx]: Please don't use React.FC (Chat-GPT??)... Write a regular TSX-function instead.

**Review by louziferr — CHANGES_REQUESTED** [review]
Changes needed

File: src/components/homepage-header.tsx (hunk 6 above)
Comment (hnns-dev) [reply src/components/homepage-header.tsx]: sven wrote this but i´ll change it!

**Pull request merged**